### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by macstewart**
- Fix broken git clone instructions
- Fix broken JMS info home link
- Make JMS library version fixed in samples and tutorials
- Merge latest template changes

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ links-java-tutorial: http://dev.solace.com/get-started/java-tutorials/topic-queu
 links-solaceCloud-setup: http://cloud.solace.com/create-messaging-service/
 docs-vmr-setup: http://docs.solace.com/Solace-VMR-Set-Up/Setting-Up-VMRs.htm
 docs-core-concepts: http://docs.solace.com/Features/Core-Concepts.htm
-docs-jms-home: http://docs.solace.com/Solace-JMS-API/JMS-home-open.htm
+docs-jms-home: https://docs.solace.com/Solace-JMS-API/jms-get-started-open.htm
```
Comment (mdspielman) [inline docs/_config.yml:18]: This doesn't seem like the final fix. It's not aligned with the // approach used in other fixes.

**Review by mdspielman — CHANGES_REQUESTED** [review]
A few changes still needed to make this a full fix.

File: docs/_config.yml (hunk 1 above)
Comment (macstewart) [reply docs/_config.yml:18]: I think the http:// vs // thing is a template issue, as I don't see any other fixes using // links in the _config.yml.

**Review by mdspielman — APPROVED** [review]

**Comment by mdspielman** [comment]
Merging this, template is now updated, so next round will improve this again.

**Pull request merged**